app/Models/Organisation.php

```diff
@@ -107,6 +108,9 @@ public function scopePreparedWith(Builder $query): Builder
                 'entity.image',
                 'location',
                 'location.entity',
+                'locations' => function ($sub) {
+                    $sub->select('id', 'name');
+                },
```

Comment:
Doesn't this need locations.entity for the tooltiped link to work without having LAZY shout at you?

Reply:
LAZY is on and I'm not getting any errors

Reply:
Are you viewing locations in the table mode (not grid)? This select is for the table mode :)